templates/scripts/create-dev-env.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+PROJECT=<% .Name %>
+AWS_DEFAULT_REGION=<% index .Params `region` %>
+RANDOM_SEED="<% index .Params `randomSeed` %>"
+ENVIRONMENT=stage  # only apply to Staging environment
+
+DEV_DB_LIST=$(aws iam get-group --group-name ${PROJECT}-developer-${ENVIRONMENT} | jq -r '"dev" + .Users[].UserName' | tr '\n' ' ')
+if [[ -z "${DEV_DB_LIST}" ]]; then
+    echo "$0: No developers available"
+    exit 2
```

Comment:
On initial `zero apply` there won't be any users yet, and this non-zero exit will cause apply to fail.

Reply:
yeah, let's exit with 0